### PR TITLE
MOTECH-2005: Added warning if http is configured as the protocol

### PR DESCRIPTION
**Pull request by akarczewski**

### CODE REVIEW

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by pgesek** [comment]
test this please

File: commcare/src/main/resources/webapp/js/controllers.js [hunk 1]
```diff
@@ -426,6 +426,14 @@
             return true;
         };
 
+        $scope.validateUrlProtocol = function () {
+            if ($scope.selectedConfig.accountConfig.baseUrl.substring(0, 8) !== "https://") {
```
Comment (sebbrudzinski) [inline commcare/src/main/resources/webapp/js/controllers.js]: You can use startsWith

File: commcare/src/main/resources/webapp/js/controllers.js [hunk 2]
```diff
@@ -426,6 +426,14 @@
...
+                return false;
+            } else {
+                return true;
+            }
+        };
```
Comment (sebbrudzinski) [inline commcare/src/main/resources/webapp/js/controllers.js:429]: All of this can be done in one line by returning the negation of the condition in if

**Comment by sebbrudzinski** [comment]
test this please

**Comment by sebbrudzinski** [comment]
test this please

**Comment by pgesek** [comment]
test this please

**Comment by pgesek** [comment]
test this please

**Comment by sebbrudzinski** [comment]
test this please

**Comment by sebbrudzinski** [comment]
@akarczewski Can you update the branch against latest master? Thanks.

**Comment by sebbrudzinski** [comment]
test this please

**Pull request merged**